src/mlpack/core/util/using.hpp

```diff
@@ -24,8 +24,11 @@ namespace mlpack {
 #ifdef MLPACK_HAS_COOT
 
 /* using for bandicoot namespace*/
-using coot::exp;
 using coot::dot;
+using coot::exp;
+using coot::find;
+using coot::find_nan;
+using coot::find_nonfinite;
```

Comment:
Do we already have these in bandicoot ?

Reply:
Yeah, they've actually been available for a good while now.